### PR TITLE
Custom styles

### PR DESCRIPTION
**Pull request by saurabhsood91**
@zebulasampedro an initial look?

### CODE REVIEW

File: share/jupyter/hub/static/less/style.less [hunk 1]
```diff
@@ -25,3 +25,4 @@
 @import "./admin.less";
 @import "./error.less";
 @import "./login.less";
+@import "./custom.less";
```
Comment (zebulasampedro) [inline share/jupyter/hub/static/less/style.less]: `custom.less` should be called `brand.less` instead.

File: share/jupyter/hub/templates/login.html [hunk 2]
```diff
@@ -22,10 +22,11 @@
   </div>
   <div class='auth-form-body'>
 
-    <p id='insecure-login-warning' class='hidden'>
+    <!-- <p id='insecure-login-warning' class='hidden'>
     Warning: JupyterHub seems to be served over an unsecured HTTP connection.
     We strongly recommend enabling HTTPS for JupyterHub.
-    </p>
+    </p> -->
+    {% include "warning.html" %}
```
Comment (zebulasampedro) [inline share/jupyter/hub/templates/login.html]: I think it is desirable to leave that warning totally unbranded to ease deployment. Remove the modifications to the SSL warning.

That said, this is the approach you should take to brand the logo/name in the top left of the page.

File: share/jupyter/hub/templates/page.html [hunk 3]
```diff
@@ -30,9 +30,10 @@ <h4 class="modal-title" id="{{key}}-label">{{title}}</h4>
     <title>{% block title %}JupyterHub{% endblock %}</title>
```
Comment (zebulasampedro) [inline share/jupyter/hub/templates/page.html]: Where do you actually change the title from _JupyterHub_ to a user-configured value?

Comment (saurabhsood91) [inline share/jupyter/hub/templates/page.html]: ah. I missed this. I will add this as well

File: share/jupyter/hub/templates/login.html (hunk 2 above)
Comment (saurabhsood91) [inline share/jupyter/hub/templates/login.html]: @zebulasampedro so, the SSL warning contains references to JupyterHub, which is why I templated it out

**Comment by saurabhsood91** [comment]
@zebulasampedro another look?

File: share/jupyter/hub/templates/page.html [hunk 4]
```diff
@@ -27,12 +27,13 @@ <h4 class="modal-title" id="{{key}}-label">{{title}}</h4>
...
 
-    <title>{% block title %}JupyterHub{% endblock %}</title>
+    <title>{% include "title.html" %}</title>
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    
+
     {% block stylesheet %}
     <link rel="stylesheet" href="{{ static_url("css/style.min.css") }}" type="text/css"/>
+    <link rel="stylesheet" href="{{ static_url("css/custom.css") }}" type="text/css"/>
```
Comment (zebulasampedro) [inline share/jupyter/hub/templates/page.html]: `brand.less` will be compiled with the rest of JupyterHub's CSS, so I'm not clear on what this line is doing.

File: config [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+    "custom_css": "/home/saurabh/workspace/jupyterhub/custom.less",
```
Comment (zebulasampedro) [inline config]: Can you commit an example `brand.less` as part of this PR?

File: share/jupyter/hub/templates/page.html (hunk 4 above)
Comment (saurabhsood91) [inline share/jupyter/hub/templates/page.html]: this is a remnant of an earlier commit. I will remove this

**Comment by saurabhsood91** [comment]
@zebulasampedro added an example `brand.less` file and removed the extra line

**Pull request closed without merging**